css/select_a_meal.css

```diff
@@ -44,7 +44,9 @@
     color:#4e4e4e;
 
 }
-
+.lab-selected{
+    background:#4e4e4e !important;
```

Comment:
I don't remember writing that... I don't even know what !important means

On Thu, Apr 16, 2015 at 4:11 PM, James Rowe <notifications@github.com>
wrote:

> In css/select_a_meal.css
> <https://github.com/FiveGuys-CS256/html-prototype/pull/76#discussion_r28554837>
> :
>
> > @@ -44,7 +44,9 @@
> >      color:#4e4e4e;
> >
> >  }
> > -
> > +.lab-selected{
> > +    background:#4e4e4e !important;
>
> There's GOTTA be a way to not use !important here :) If the background
> color is not getting shown, use a more specific query selector and you
> won't need to use important. If you don't care to make that change I'm not
> insulted though. !important is just considered bad practice.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FiveGuys-CS256/html-prototype/pull/76/files#r28554837>
> .
>